spec.html

```diff
@@ -8353,6 +8353,28 @@ <h1>ECMAScript Function Objects</h1>
             *true* if this is a strict function, *false* if this is a non-strict function.
           </td>
         </tr>
+        <tr>
+          <td>
+            [[PresentInStackTraces]]
+          </td>
+          <td>
+            Boolean
+          </td>
+          <td>
+            When *false*, this function must not have a corresponding entry in stack traces.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            [[HasSourceTextAvailable]]
+          </td>
+          <td>
+            Boolean
+          </td>
+          <td>
+            When *false*, this function will be rendered as a |NativeFunction| by `Function.prototype.toString` (<emu-xref href="#sec-function.prototype.tostring"></emu-xref>) and corresponding stack trace entries will have no attribution or position information.
```

Comment:
Not an editorial review comment, but, I'd prefer "have no attribution or position information" to be replaced with "have implementation-defined non-incidental attribution or position information", using the definition of incidental given below.

Reply:
How about just "... will have no incidental attribution or position information, as defined in (section reference)."?

Reply:
That sgtm.